xml/rmt_install.xml

```diff
@@ -31,7 +33,232 @@
    possible for a single server to host both simultaneously.
   </para>
  </warning>
+ <sect1 xml:id="sec-rmt-storage-requirements">
+     <title>Storage Requirements</title>
+     <para>
+         Downloaded packages are stored in <filename>/var/lib/rmt/public/repo/SUSE/Products/</filename>. 
+         The amount of storage your &rmt; server requires is dependent on several variables: the 
+         number of  repositories and architectures that you mirror, how many packages are 
+         downloaded, the  number of  extensions that are enabled, and the age of your server installation. 
+         Outdated  packages are not removed by &rmt;, but accumulate over time. When new versions are 
+         downloaded, the old versions are kept on disk but are not referenced by repository 
+         metadata, which makes them invisible to clients.
+     </para>
+     <para>
+         As a general guide, two to three times the total size of all mirrored repositories should be 
+         sufficient. If you are mirroring a single release, such as &slea; 15.1, 250 GB provides room
+         for packages and updates. The total size of all x86_64 repositories for &slea; 12 SP4 is about 130 GB, and about   
+         155 GB for &slea; 15. There is not (yet) a tool for reporting repository sizes, so it is a good practice to monitor disk 
+         space and plan for possible expansion. The following table lists examples of some approximate repository sizes for 
+         &slea; 12 and 15, comparing the size of the repositories that are enabled by default with the total available in each repository.
+     </para>
+     <table xml:id="sec-rmt-repository-sizes">
+   <title>Example Repository Sizes</title>
+   <tgroup cols="3" align="left">
+        <thead>
+     <row>
+      <entry>
+       <para>
+        Product
+       </para>
+      </entry>
+      <entry>
+       <para>
+        Size of default repositories in GB
```

Comment:
They are called `mandatory` rather than `default` elsewhere in the docs: https://github.com/SUSE/doc-sle/blob/30ba9ed801005014c5fe2bfa9340f89fc0cd69e5/xml/rmt_rmt-cli.xml#L125